entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2155
```

Comment:
I've had issues with this in the past... it's not a good idea when using `set -e`. I don't recommend it. I know that it makes a bit clumsy to declare locals or export variables, but it's a good check to have.

Reply:
fixed in 72b6868